File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonBlue.java
package org.firstinspires.ftc.teamcode;

import com.bylazar.configurables.annotations.Configurable;
import com.pedropathing.geometry.Pose;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
import org.firstinspires.ftc.robotcore.external.navigation.DistanceUnit;
import org.firstinspires.ftc.robotcore.external.navigation.Pose2D;

@Configurable
@TeleOp(name = "BLUE Auton")
public class AutonBlue extends AutonBase {
    public static Config c = new AutonBase.Config(new Pose(79.170, 18.581, 90),new Pose(81.997, 81.391),new Pose(84.623, 34.738),new Pose(84.623, 60.387), new Pose(84.623, 80.785),new Pose(42.6143057504,0 ),new Pose(81.795, 81.593), new Pose2D(DistanceUnit.INCH, 12.926, 135.921, AngleUnit.DEGREES, 60));
    @Override
    Config getConfig() {
        return c;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonBlue.kt
package org.firstinspires.ftc.teamcode

import com.pedropathing.geometry.Pose
import com.qualcomm.robotcore.eventloop.opmode.Autonomous

@Autonomous(name = "BLUE Auton")
class AutonBlue : Auton() {
    override fun getWaypoints(): Waypoints {
        return w
    }

    companion object {
        @JvmField
        var w =
            Waypoints(
                Pose(88.0, 8.0, 90.0),
                Pose(102.0, 36.0, 0.0),
                Pose(0.0, 24.0, 0.0),
                Pose(18.0, 0.0, 0.0),
                Pose(82.0, 96.0, 150.0)
            )
    }
}

